src/Data/List/NonEmpty.purs

```diff
@@ -133,6 +188,14 @@ toList (NonEmptyList (x :| xs)) = x : xs
 singleton :: forall a. a -> NonEmptyList a
 singleton = NonEmptyList <<< NE.singleton
 
+-- | An infix synonym for `range`.
+infix 8 range as ..
+
+-- | Create a list containing a range of integers, including both endpoints.
+-- Todo, rewrite this without unsafe workaround (if necessary)
+range :: Int -> Int -> NonEmptyList Int
+range start end = unsafePartial fromJust $ fromList $ L.range start end
```

Comment:
Perhaps this should be rewritten so that the `start` arg is always included in the range and guarantees the NonEmpty part of the returned value, so we don't need to wrap it in a `Maybe`?

Reply:
I think the second one looks good.